tests/requirements.txt

```diff
@@ -13,3 +13,5 @@ selenium
 flake8
 pylint
 pytest-dash>=1.0.1
+dash-cytoscape
```

Comment:
do you need this? the tests will do a `import dash_cytoscape` which will do a local import of the package. If that local import doesn't work, then I'd be worried that this package install would install the package from pip rather than what is included in this PR.

Reply:
Will do!